shared/gh/partials/tenant-student/student-terms-and-conditions-modal.html

```diff
@@ -0,0 +1,20 @@
+<div class="modal fade" id="gh-terms-and-conditions-modal" tabindex="-1" role="dialog" aria-labelledby="modal-title" aria-hidden="true">
+    <div class="modal-dialog">
+        <div class="modal-content">
+            <div class="modal-header">
+                <button type="button" class="close btn-reverse" data-dismiss="modal">
+                    <span aria-hidden="true">×</span>
+                    <span class="sr-only">Close</span>
+                </button>
+                <h4 class="modal-title">Terms and Conditions</h4>
+            </div>
+            <div class="modal-body">
+                <%= data.text %>
```

Comment:
It would be neat if the T&C could be provided in markdown format? (At the very minimum we should probably convert line-breaks to paragraphs)